licenses/management/commands/check_for_translation_updates.py

```diff
@@ -1,8 +1,20 @@
-from django.core.management import BaseCommand
+from django.core.management import BaseCommand, call_command
 
-from licenses.transifex import check_for_translation_updates
+from licenses.transifex import TransifexHelper
+
+TOP_BRANCH = "develop"
+
+branch_name = "test_branch"
 
 
 class Command(BaseCommand):
     def handle(self, **options):
-        check_for_translation_updates()
+        branches_updated = TransifexHelper().check_for_translation_updates()
+
+        # run collectstatic if we're going to publish
```

Comment:
Should we log when we are running collectstatic and that we are publishing updated branches?